runners/direct-java/src/main/java/org/apache/beam/runners/direct/WindowEvaluatorFactory.java

```diff
@@ -123,8 +125,8 @@ public Instant timestamp() {
     }
 
     @Override
-    public Collection<? extends BoundedWindow> windows() {
-      return value.getWindows();
+    public BoundedWindow window() {
+      return Iterables.getOnlyElement(value.getWindows());
```

Comment:
The evaluator doesn't currently explode input windows, so this will break for any element that enters in multiple windows - I've authored #313 to cover this